codes/plotting/gen_fig4.py

```diff
@@ -42,6 +48,10 @@
 }
 
 
+start_date = "2020-01-15"
+end_date = "2020-03-18"
```

Comment:
Is there a reason we are stopping at 3/18? No worries if that was decided, but since we're updating things i was figuring we would go further in the figure as well? Also, we should probably pull these dates from `cutoff_dates.csv` rather than hardcoded if possible (at least the end dates)? Let me know if you think theres a reason for them to be hardcoded

Reply:
took care of this, didnt realize that csv existed!